.github/workflows/ci.yml

```
@@ -12,297 +10,319 @@
+    
+    - name: Install frontend dependencies
+      working-directory: frontend
+      run: bun install
+    
+    - name: Run Go unit tests (parallel)
+      run: go test ./... -v -parallel 4 -count=1
+    
+    - name: Run frontend unit tests
+      working-directory: frontend
+      run: bun run test --run
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)